### ISSUE TITLE
New Kudos Artwork

### ISSUE THREAD

**Issue by fuguefoundation** [gitcoinco/kudosbadges#297]
# Timeline

About two weeks is fine.

# Title

Fugue Foundation

# Design Inspiration

1. Please use [this logo](https://github.com/fuguefoundation/ff-dapp/blob/master/src/assets/images/logo_1500.png) as the design the kudos. I've seen some of the animated kudos (example [1](https://gitcoin.co/kudos/20556/infinicube), example [2](https://gitcoin.co/kudos/20554/psychedelic_color)) and would like to incorporate something like that into the one created here. Here's what I'm thinking.

* So the symbol used in the logo is a "F hole", commonly seen on the body of violins and cellos. Can you have the kudos begin with the two F holes facing the same direction (so it looks like F F), then spin/rotate the second F to have it appear like it does in the logo with the two Fs crossing at the top? 

2. Along the hexagon edge on the bottom left, put the word "Fugue", and on the bottom right of the hexagon, put "Foundation" (white text over black).

3. Colors used:
* Red: #b31121
* Black/White Gradient: #000000 to #ffffff to #000000

4. Not sure how this usually works, but maybe the artist could prepare a draft version and then ping me to make sure we're on the same page before finalizing? Please let me know if anything here doesn't make sense, happy to provide more clarity or proofing.

# Down Payment

https://etherscan.io/tx/0x69c60b4c45130ce9e5531be97b0083ad57fbf2c157a9dbc541bcc51c983b6012

# Gen0 Owner Address

0x35DF44e9ac44943c03A14dD276Dc03981ad7C869

**Comment by owocki** [comment]
@MladenPetronijevic u want this one

**Comment by MladenPetronijevic** [comment]
@owocki i would take it, but i'm not sure how those animated kudos are made.
maybe the original creator of those kudos could give it a try, if not maybe i could ask him/her how it's made and then try making it

**Comment by owocki** [comment]
got it ; @l-kh in that case do u want it?

**Comment by L-KH** [comment]
@owocki , Thanks for inviting me. I am working on it.
@MladenPetronijevic  , I am creating it using SVGator, its online service but you need first to create your full SVG first before import it for animation. I am here for questions.

**Comment by L-KH** [comment]
@fuguefoundation , Please check those examples and let me know about what do you think.
[FugueFoundation_Kudos.zip](https://github.com/gitcoinco/kudosbadges/files/4918905/FugueFoundation_Kudos.zip)

◉ I use the font "Apparatus SIL" because it seems like the one from the website.
◉ Hexagone color is "0A0A0A" let me know if you want to be changed

**Comment by fuguefoundation** [comment]
@L-KH examples look good so far, I like the Apparatus SIL font. Let's move the word "Fugue" to the top left part of the hexagon. Thanks1

**Comment by L-KH** [comment]
@fuguefoundation , which example that seem good to you? idea1 or idea2?

check this one, and let me know for any modifications
[FUGUEFOUNDATION.zip](https://github.com/gitcoinco/kudosbadges/files/4919769/FUGUEFOUNDATION.zip)

**Comment by fuguefoundation** [comment]
@L-KH Text placement/font looks good in the new version. In terms of animation, I like version 2 from the first group you sent, but with a few small adjustments:
1. Have the F-holes pause longer (about two seconds) for both the F F and when the two Fs cross to form the logo
2. Make the movements (slides, rotations) faster
3. Fast fade in/out of the F at the beginning and end of the animation sequence

We're really close, thanks!

**Comment by L-KH** [comment]
@fuguefoundation , can please check this one and let me know if there are any modifications needed.
ps: is this the version 2 that you mentioned? 
[FugueFoundation5.zip](https://github.com/gitcoinco/kudosbadges/files/4924960/FugueFoundation5.zip)

here is the timing used:
0.5s movement position
0.5s rotation
0.5s fade-in/out
1.5s pause
0.5s empty

**Comment by fuguefoundation** [comment]
@L-KH Yes, that's the right version. Let's try this timing and this modification, then it should be good to go:
0.2s first slide movement
3s pause F F
0.2s rotation
5s pause crossed Fs (the original logo)
fade out (so remove the final rotation, slide movement to the right side, just fade out once you create the image of the original logo)

**Comment by L-KH** [comment]
sorry I am not sure if I really understand what you mean, but can u please check this one !

[FugueFoundation6.zip](https://github.com/gitcoinco/kudosbadges/files/4926392/FugueFoundation6.zip)

here is the timing used
0.5s fade in
0.2s movement position F
3s pause
0.2s rotate
5s pause
0.5s fade out

let me know for any modifications needed

**Comment by fuguefoundation** [comment]
@L-KH you got it, looks great! One thing though, you changed the font to something different on this one. Please put it back to what it was (Apparatus SIL) and we're done. Great job!

**Comment by L-KH** [comment]
@fuguefoundation , Oh sorry my bad. I didn't notice the font, you can check this one:
[FugueFoundation7.zip](https://github.com/gitcoinco/kudosbadges/files/4926784/FugueFoundation7.zip)
Thanks

**Comment by fuguefoundation** [comment]
@L-KH confirmed, thanks! Tagging @owocki as well for situational awareness in case he needs to do something for payment.

**Comment by owocki** [comment]
if you are happy with the work @fuguefoundation then i can just send @L-KH the $$ u sent me

btw can someone submit a gitcoin.co/kudos/new so i can mint em?

**Comment by fuguefoundation** [comment]
Thanks @owocki , good to go for payment. I will be submitting the mint request in the next few days.

**Comment by L-KH** [comment]
received, THank you all.
@fuguefoundation , here is my profile for any future modifications/works: https://gitcoin.co/l-kh
thanks
